member-service/src/main/java/com/hedvig/memberservice/services/customerio/CustomerIOEventListener.kt

```diff
@@ -0,0 +1,119 @@
+package com.hedvig.memberservice.services.customerio
+
+import com.hedvig.integration.notificationService.NotificationService
+import com.hedvig.memberservice.events.EmailUpdatedEvent
+import com.hedvig.memberservice.events.NameUpdatedEvent
+import com.hedvig.memberservice.query.MemberEntity
+import com.hedvig.memberservice.query.MemberRepository
+import com.hedvig.memberservice.services.trustpilot.TrustpilotReviewService
+import com.hedvig.resolver.LocaleResolver
+import com.neovisionaries.i18n.CountryCode
+import org.axonframework.config.ProcessingGroup
+import org.axonframework.eventhandling.EventHandler
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.context.annotation.Profile
+import org.springframework.stereotype.Component
+import java.util.Locale
+import java.util.Objects
+
+@Component
+@Profile("customer.io")
+@ProcessingGroup("SegmentProcessorGroup")
+class CustomerIOEventListener @Autowired constructor(
```

Comment:
`@Autowired constructor` is not needed but it's always added when converting from java
```suggestion
class CustomerIOEventListener(
```